### ISSUE TITLE
support for deployment to server using QRS

### ISSUE THREAD

**Issue by Marnnn** [stefanwalther/sense-go#39]
Hi Stefan,

I noticed in the code for this tool you currently have a note next to the QRS setup saying yet to be implemented. I was wondering how you are getting on with this? Is it possible, or is it something that could be a long way off being included. Nice job on the tool already though!

Thanks

**Comment by stefanwalther** [comment]
Hi,

first of all I am happy that you like sense-go. I am curious about how you use it. In general feedback for sense-go is that it is so hard to set-up and understand ...

Regarding QRS integration:
- I actually started to think of it a long time ago
- All pieces are already there
  - See qrs: https://github.com/stefanwalther/qrs/
  - And in particular https://github.com/stefanwalther/qrs/blob/master/lib/sugar/ep-extension.js
- So what needs to be done is to add `qrs` as a dependency and write a gulp task ...

I am not sure when and whether I will find time to provide that, but I am more than happy to evaluate and help with a PR?!

**Comment by Marnnn** [comment]
I actually hit a few issues with it initially and there are a few new things in there for me to understand so I took the opportunity to learn yeoman and I've gone with my own setup at the moment, one that massively takes inspiration from this and your yeoman generator. My generator includes a gulp with a bunch of tasks I've written to get my own workflow. I really like the idea of using this and it being something improved by the community but also want more control and ensure I'm understanding everything that's happening.

The next Gulp task I'm going to write is for deploying via the QRS, I've had some success with using qrs-interact from eapowertools so that should be happening next week I hope. When I've successfully implemented this I'll look at including what I've done into here.
